README.md

```diff
@@ -121,6 +121,7 @@ Rules with a check mark (✅) are enabled by default while using the `plugin:cyp
 | ✅  | [no-assigning-return-values](./docs/rules/no-assigning-return-values.md)   | Prevent assigning return values of cy calls                     |
 | ✅  | [no-unnecessary-waiting](./docs/rules/no-unnecessary-waiting.md)           | Prevent waiting for arbitrary time periods                      |
 | ✅  | [no-async-tests](./docs/rules/no-async-tests.md)                           | Prevent using async/await in Cypress test case                  |
+| ✅  | [unsafe-to-chain-command](./docs/rules/unsafe-to-chain-command.md)         | Prevent chaining from unsafe to chain commands                  |
```

Comment:
This wasn't added to the [recommended rules](https://github.com/cypress-io/eslint-plugin-cypress/blob/master/lib/config/recommended.js), so it shouldn't be marked as enabled by default.

Reply:
Or should be added to the recommended rules.

Reply:
Added as a new PR: https://github.com/cypress-io/eslint-plugin-cypress/pull/119